docs/index.md

```diff
@@ -32,3 +32,8 @@
 
    + [Custom Devices Tips & Tricks](Custom_Device_Tips_and_Tricks.md)
    + [Scripting APIs for Custom Devices](Scripting_API.md)
+
+#### Migrating from an LLB to a PPL based Custom Device
+
+   + [General Information](Migrating_LLB_to_a_PPL_based_Custom_Device.md)
```

Comment:
Please change the section title from "General Information" to "Migrating a Custom Device from LLB to PPL"